src/golang/lib/livy/entrypoints.go

```diff
@@ -0,0 +1,75 @@
+package livy
+
+const (
+	InstallAqueductEntrypoint = `import sys
+import subprocess
+
+print(1)
+subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'aqueduct-ml==0.2.1'])
+`
+
+	FunctionEntrypoint = `import base64
```

Comment:
How are these entry points being used?

Reply:
These entrypoints are the base code that are being shipped to Spark. We inject the spec into the entrypoint before shipping.